test/task.h

```diff
@@ -184,6 +184,7 @@ enum test_status {
 #  define inline __inline
 # endif
 
+# if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
Slightly related: can we also check `inline` keyword as `ifndef inline` here: https://github.com/equalsraf/libuv/blob/tb-msvc2015/test/task.h#L183.
I think VS2015 (and maybe VS2013) both support it.